### PR TITLE
himalaya: generate completions

### PR DESCRIPTION
**Pull request by nikaro**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/himalaya.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Himalaya < Formula
   url "https://github.com/soywod/himalaya/archive/v0.7.1.tar.gz"
   sha256 "79ad75e765a5298c5c597beb175cec1d2982eead2172bc78daa7a9f159c99861"
   license "MIT"
+  revision 1
```
Comment (ZhongRuoyu) [inline Formula/himalaya.rb:7]: A revision bump isn't necessary for completions, as it implies that users will have to upgrade their installation with `brew upgrade`. That would be too much for a small improvement.

**Review by p-linnane — APPROVED** [review]
Thanks for your first contribution to `homebrew-core` @nikaro!

File: Formula/himalaya.rb (hunk 1 above)
Comment (nikaro) [reply Formula/himalaya.rb]: I understand, the idea was to not wait for the next release of Himalaya for having completions generated by brew at installation.

Comment (ZhongRuoyu) [reply Formula/himalaya.rb]: I get what you mean. In that case, anyone wishing to use the completions may do a `brew reinstall`.

**Pull request closed without merging**